workspace/__lib__/xod/color/format-color/patch.cpp

```diff
@@ -0,0 +1,30 @@
+
+struct State {
+    char str[7] = { 0, 0, 0, 0, 0, 0, '\0' };
+    CStringView view;
+    State() : view(str) { }
+};
+
+// clang-format off
+{{ GENERATED_CODE }}
+// clang-format on
+
+char nibbleToChar(uint8_t nibble) {
+  return (nibble < 10)
+    ? '0' + nibble
+    : 'a' + nibble - 10;
```

Comment:
```suggestion
    : 'A' + nibble - 10;
```

Node description says we’ll have uppercase letters